docs/source/migration/3.0.mdx

```diff
@@ -323,52 +364,22 @@ class GetHero(val id: Input<String?> = Input.absent())
 class GetHero(val id: String?)
 ```
 
-By default the [GraphQL spec](https://spec.graphql.org/draft/#sec-Coercing-Variable-Values) treats variables of nullable type as optional. It is valid to omit them at runtime. In practice though, this is rarely used and makes the operations declaration verbose.
+By default, the [GraphQL spec](https://spec.graphql.org/draft/#sec-Coercing-Variable-Values) treats nullable variables as optional, so it's valid to omit them at runtime. In practice, however, this is rarely used and makes the operation's declaration verbose.
 
-Apollo Android 3.x removes the `Optional` wrapper so that it's easier to construct a query having nullable variables.
+Apollo Android 3.x removes the `Optional` wrapper so that it's more straightforward to construct a query with nullable variables.
 
-If omitting a variable is desired, it's possible to opt-in the `Optional` wrapper with the `@optional` directive:
+> **Note**: This new behavior applies _only_ to variables. Fields of input objects still use `Optional`, because it's common to omit particular input fields.
```

Comment:
Nitpick: maybe remove the "Note" here since we have the quotes highlighting the block already? 

```suggestion
> This new behavior applies _only_ to variables. Fields of input objects still use `Optional`, because it's common to omit particular input fields.
```